go/contacts/tasks.py

```diff
@@ -33,10 +33,13 @@ def delete_group(account_key, group_key):
     # We do this one at a time because we're already saving them one at a time
     # and the boilerplate for fetching batches without having them all sit in
     # memory is ugly.
-    for contact_key in group.backlinks.contacts():
-        contact = contact_store.get_contact_by_key(contact_key)
-        contact.groups.remove(group)
-        contact.save()
+    contacts_page = group.backlinks.contact_keys()
+    while contacts_page is not None:
+        for contact_key in contacts_page:
+            contact = contact_store.get_contact_by_key(contact_key)
+            contact.groups.remove(group)
+            contact.save()
+        contacts_page = contacts_page.next_page()
```

Comment:
We'd have to use something like the `PausingDeferredQueue` in go-api-toolkit, but the extra layers of juggling there probably aren't worth it for the small reduction in boilerplate we'd get. Either way, it's out of scope for this PR.